src/NuGet.Config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@akoeplinger Thanks for that. I'm using Git for Windows, and git diff HEAD~..HEAD -- src/NuGet.Config wasn't showing the BOM difference. :unamused: